function add(a,b){
    return new Promise((res,rej)=>{
        const sum=a+b;
        res(sum);
    })
}
add(5,6)
.then((res)=>{
    console.log(res);
})
.catch((error)=>{
    console.log(error);
})






function add(a,b){
    return new Promise((res,rej)=>{
        const sum=a+b;
        res(sum);
    })
}

add(5,6)
.then((res)=>{
    console.log(res);
})
.catch((error)=>{
    console.log(error);
})